src/pages/MapViewer/styles.scss

```diff
@@ -35,6 +35,25 @@
   }
 }
 
+.qa-view-icon {
+  position: absolute;
+  top: 58.5px;
```

Comment:
Is there some subpixel rendering bug?

Reply:
I don't know what you mean, the `Data Catalogue` button starts at `8.5px` and I wanted any buttons below to be `50px` apart. This is the only reason why I have used half pixels.